aiormq/exceptions.py

```diff
@@ -136,9 +139,20 @@ class MethodNotImplemented(AMQPError):
 
 
 class DeliveryError(AMQPError):
-    __slots__ = 'message', 'frame'
+    __slots__ = 'returned_message', 'frame'
+
+    message = '%s: %s'
 
-    def __init__(self, message, frame):
-        self.message = message
+    def __init__(self, frame: spec.Frame,
```

Comment:
Swapping of this arguments breaks public API. Why it's so important?

Reply:
It's not important, just an improvement. `frame` is required parameter and `returned_message` is optional so I moved it to the end. I could revert this change.